pages/index.js

```diff
@@ -437,10 +437,9 @@ const Page = () => (
             name="Curriculum"
             desc={
               <>
-                We’re dropping a new{' '}
-                <a href="https://workshops.hackclub.com/">workshop</a> every
-                Monday afternoon, and you can join our Protoclub meeting to see
-                how to run it live.
+                Come prepared to every meeting with over 100{' '}
```

Comment:
Something doesn't feel right about this one tbh: idk but it sounds like you are coming to every meeting with over 100 workshops in my head lol?

Reply:
How about splitting it into two sentences?

```
We've got over 100 workshops that you can deliver with ease.  Each of them guides your club members through making a unique project.
```